pki.go

```diff
@@ -237,6 +237,8 @@ func loadCAPoolFromConfig(l *logrus.Logger, c *config.C) (*cert.NebulaCAPool, er
 			return nil, errors.New("no valid CA certificates present")
 		}
 
+	} else if errors.Is(err, cert.ErrInvalidPEMCertificateUnsupported) {
```

Comment:
We should ensure there is at least 1 valid CA in the pool otherwise it won't work

Reply:
`ErrInvalidPEMCertificateUnsupported` never get added to the pool. Do we not already have semantics that catch size-zero pools? I'll check.

Reply:
We do a few lines up. `caPool, err := cert.NewCAPoolFromBytes(rawCA)` will continue parsing the provided bytes in the event of a `ErrInvalidPEMCertificateUnsupported` but if all are too new (or expired) and we hit this branch then we can swap in a `CAPool` that has no valid certificates.

Reply:
We did not. Added.